src/style-sheets/responsive/_screens.scss

```diff
@@ -1,5 +1,6 @@
 // Smaller mobile devices and portrait phones
-$screen-xs: 0;
+$screen-xxs: 0;
+$screen-xs: 415px;
```

Comment:
so i like the idea but our break points were set by our UX design team so we can't really stray from them. The carousel was one of our old components that we don't use at the moment so we can move the media query but should just keep it hard coded there. Could you revert the variable for screen-xs?

Reply:
Sure!